modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs

```diff
@@ -461,7 +467,7 @@ public static bool IsZeroApprox(real_t s)
         /// <returns>The resulting value of the interpolation.</returns>
         public static real_t Lerp(real_t from, real_t to, real_t weight)
         {
-            return from + ((to - from) * weight);
+            return from + (to - from) * weight;
```

Comment:
IIRC the additional parenthesis were added on purpose with a formatting option for explicit parenthesis.

Reply:
I find parentheses strange to separate multiplication from addition, but if it's a convention, ok.